elections/steering/2025/README.md

```diff
@@ -262,8 +260,7 @@ The Steering Committee has selected the following people as [election officers]:
 In addition, the following contributors are helping with the election:
 
 - Alternate Officers: Rey Lejano, Arujjwal Negi
-- Infra Liaison: TBD
-- Contributor Comms Liaison: TBD
+- Infra Liaison: Mahamed Ali
```

Comment:
aside: should we link to github profiles or something like that?
also thanks @upodroid

Reply:
I would volunteer for comms :)

Reply:
I tried tagging them in the file, but GitHub doesn't render the usernames in the preview mode. 
https://github.com/kubernetes/community/blob/32dda65512e649dd9a527bf0e8eb15a3519964bb/elections/steering/2025/README.md?plain=1#L262-L264